nautobot_data_validation_engine/custom_validators.py

```diff
@@ -146,6 +148,41 @@ def clean(self):
                     continue
                 compliance_class(self.context["object"]).clean()
 
+        if not exclude_disabled_rules:
+            if rule:
```

Comment:
When trying to run, I get the following error here:

![image](https://github.com/nautobot/nautobot-app-data-validation-engine/assets/111259311/d421a0ff-7fe8-403f-9f7b-0babdac337cd)

```
ERROR/ForkPoolWorker-6] Task nautobot_data_validation_engine.jobs.RunRegisteredDataComplianceRules[e967e127-5b9e-40e2-9a91-ddbc40bcd96c] raised unexpected: UnboundLocalError("cannot access local variable 'rule' where it is not associated with a value")
nautobot-data-validation-engine-worker-1    | Traceback (most recent call last):
nautobot-data-validation-engine-worker-1    |   File "/usr/local/lib/python3.11/site-packages/celery/app/trace.py", line 477, in trace_task
nautobot-data-validation-engine-worker-1    |     R = retval = fun(*args, **kwargs)
nautobot-data-validation-engine-worker-1    |                  ^^^^^^^^^^^^^^^^^^^^
nautobot-data-validation-engine-worker-1    |   File "/usr/local/lib/python3.11/site-packages/nautobot/extras/jobs.py", line 148, in __call__
nautobot-data-validation-engine-worker-1    |     return self.run(*args, **deserialized_kwargs)
nautobot-data-validation-engine-worker-1    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
nautobot-data-validation-engine-worker-1    |   File "/source/nautobot_data_validation_engine/jobs.py", line 81, in run
nautobot-data-validation-engine-worker-1    |     self.report_for_validation_rules()
nautobot-data-validation-engine-worker-1    |   File "/source/nautobot_data_validation_engine/jobs.py", line 118, in report_for_validation_rules
nautobot-data-validation-engine-worker-1    |     validator(validated_object).clean(exclude_disabled_rules=False, logger=logger)
nautobot-data-validation-engine-worker-1    |   File "/source/nautobot_data_validation_engine/custom_validators.py", line 152, in clean
nautobot-data-validation-engine-worker-1    |     if rule:
nautobot-data-validation-engine-worker-1    |        ^^^^
nautobot-data-validation-engine-worker-1    | UnboundLocalError: cannot access local variable 'rule' where it is not associated with a value
```

Reply:
@DistantVoyager It should be good now.